### PR TITLE
New spec to get if SCA is enabled on satellite

### PR DESCRIPTION
**Pull request by huali027**
Signed-off-by: Huanhuan Li <huali@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [x] Is this PR an enhancement?

### CODE REVIEW

File: insights/parsers/satellite_postgresql_query.py
Comment (xiangce) [inline insights/parsers/satellite_postgresql_query.py:233]: Since it's said `If the value of content_access_mode is "org_environment", it means the SCA is enabled for this organization`. 
I think it should be an `attribute` of the `"Organization"` returned by the SQL.

Or, it should be `If one of the organizations is SCA enabled, the Satellite is SCA enabled`, right

**Review by xiangce — CHANGES_REQUESTED** [review]

File: insights/parsers/satellite_postgresql_query.py
Comment (huali027) [reply insights/parsers/satellite_postgresql_query.py:233]: @xiangce, yes, the latter one is right. Let me update the comment to make it more clear.

**Review by xiangce — APPROVED** [review]
Approved pending the approval of spec

**Pull request merged**
